README.md

```diff
@@ -1,96 +1,64 @@
 # KubeVirt Ansible
 
-This repository provides collection of playbooks to
-- [x] [Install KubeVirt on existing OpenShift cluster](#install-kubevirt-on-existing-cluster)
-- [ ] Deploy Kubernetes cluster on given machines and install KubeVirt
-- [x] [Deploy OpenShift cluster on given machines and install KubeVirt](#deploy-kubernetes-or-openshift-and-kubevirt)
-- [x] [Provision resources, deploy cluster and install KubeVirt](#deploy-new-kubernetes-or-openshift-cluster-and-kubevirt-with-lago)
+This repository provides a collection of playbooks to
+- [x] [Install KubeVirt on an existing OpenShift cluster](#install-kubevirt-on-existing-cluster): install-kubevirt-on-openshift.yml
+- [ ] Deploy a Kubernetes cluster on given machines and install KubeVirt: deploy-kubernetes.yml
+- [x] [Deploy an OpenShift cluster on given machines and install KubeVirt](#deploy-kubernetes-or-openshift-and-kubevirt): deploy-openshift.yml 
+- [x] [Provision resources, deploy a cluster and install KubeVirt](#deploy-new-kubernetes-or-openshift-cluster-and-kubevirt-with-lago): control.yml
 
 *NOTE: Checked box means that playbook is working and supported, unchecked box means that playbook needs stabilization.*
 
 **Tested on CentOS Linux release 7.4 (Core), OpenShift 3.7 and Ansible 2.4.2**
 
-## List of playbooks
-
-| Playbook name | Description |
-| ------------- | ----------- |
-| install-kubevirt-on-openshift.yml | This playbook installs KubeVirt on existing OpenShift cluster. |
-| deploy-kubernetes.yml | This playbook deploys Kubernetes cluster on given machines. |
-| deploy-openshift.yml | This playbook deploys OpenShift cluster on given machines. |
-| control.yml | This is top level playbook which encapsulates entire flow, provision resources, deploying cluster, and installing KubeVirt on it. |
-
 ## Requirements
 
 Install depending roles, and export `ANSIBLE_ROLES_PATH`
 
 ```bash
+$ mkdir $HOME/galaxy-roles
+$ curl -O https://raw.githubusercontent.com/kubevirt/kubevirt-ansible/master/requirements.yml
 $ ansible-galaxy install -p $HOME/galaxy-roles -r requirements.yml
 $ export ANSIBLE_ROLES_PATH=$HOME/galaxy-roles
 ```
 
-For OpenShift deployment clone [**OpenShift Ansible project**][openshift-ansible-project]
+For OpenShift deployment clone [**OpenShift Ansible**][openshift-ansible-project]
 
 ```bash
 $ git clone -b release-3.7 https://github.com/openshift/openshift-ansible
 ```
 
-## (Optional) Setup a cluster
-
-This section describes how to setup cluster on given machines. There are two
-playbook which you can use depends on what cluster you want to bring up.
+## Set up a cluster
```

Comment:
We should highlight that this is optional if you alreday have a cluster running.